mlflow/utils/async_logging/run_operations.py

```diff
@@ -1,16 +1,11 @@
 class RunOperations:
-    """
-    Represents a collection of operations on one or more MLflow Runs, such as run creation
-    or metric logging.
-    """
+    """Class that helps manage the futures of MLflow async logging."""
 
     def __init__(self, operation_futures):
         self._operation_futures = operation_futures or []
 
     def wait(self):
-        """
-        Blocks on completion of the MLflow Run operations.
-        """
+        """Blocks on completion of all futures."""
```

Comment:
I'm not a big fan of this style. I think:


```
"""
foo
"""
```

is readable than:

```
"""foo"""
```

Reply:
The second line is the Google style guide, so here is how it works:
- for one-line docstring: """high-level description"""
- for multi-line: 
    """high-level description

    detailed description
    """

Reply:
still not a big fan tho because:

1. I need to wonder "Should I provide descriptions or not?"
2. If yes, I need to think about two descriptions, high-level and detailed.
3. I need to review two descriptions.

Reply:
actually both Numpy and Google style guide use this style. To me I usually find the high-level description pretty useful. detailed description is only required when it cannot be covered by one sentence.

Reply:
Is a high-level description always required?

Reply:
I'm fine if we don't enforce high-level description when detailed one is provided in the future.

Reply:
I just don't like the codebase would look like this:

```
def f():
    """foo"""

def f2():
    """
    foo
    """


def f3():
    """foo
    
    bar
    """
```

Reply:
This is just my preference. You can also ask other team members.

Reply:
discussed offline - we don't enforce it for now, and let's try being consistent within each file.